spec/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser_spec.rb

```diff
@@ -16,71 +30,60 @@
     ems
   end
 
-  let(:refresh_parser) do
-    VCR.use_cassette("#{described_class.name.underscore}_aicc") { described_class.new(ems) }
+  let(:ems_inv_to_hashes) { described_class.new(ems).ems_inv_to_hashes }
```

Comment:
Using the `let` statement here the `ems_inv_to_hashes` will be performed for each `it` present in your tests, what I think that is not your intention, so I will recommend you to save the `ems_inv_to_hashes` result in a instance variable that will be available for all your tests. Once you are not using the `:auth` and `:ems` variables in any place but in the `ems_inv_to_hashes` execution, maybe you will want to do something like this:
```ruby
before(:all) do
    auth = FactoryGirl.create(:authentication,
                              :userid   => "admin",
                              :password => "password",
                              :authtype => "default")

    ems = FactoryGirl.create(:physical_infra,
                             :name      => "LXCA",
                             :hostname  => "10.243.9.123",
                             :port      => "443",
                             :ipaddress => "https://10.243.9.123/443")
    ems.authentications = [auth]
    @result = described_class.new(ems).ems_inv_to_hashes
  end
```